cpp/src/arrow/csv/reader.cc

```diff
@@ -254,22 +271,12 @@ class ThreadedBlockReader : public BlockReader {
  public:
   using BlockReader::BlockReader;
 
-  static Iterator<CSVBlock> MakeIterator(
```

Comment:
I presume this was unused?

Reply:
Yes. I think it was in just for symmetry with BlockReader